server2/Arcadia.Assistant/Arcadia.Assistant.Notifications.Contracts/Models/NotificationType.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Arcadia.Assistant.Notifications.Contracts.Models
+{
+    public enum NotificationType
```

Comment:
why is it in a shared library? Notification type should not be used outside of this service

Reply:
Moved to lib